.travis.yml

```diff
@@ -46,10 +46,8 @@ script:
   - COVERALLS_REPO_TOKEN=vKN3jjhAOwxkv9HG0VBX4EYIlWLPwiJ9d npm test
   - npm run test-e2e
   # Test fxa-auth-mailer
-  - cd mailer
   - grunt templates
   - npm test
```

Comment:
We already run `npm test` with `COVERALLS_REPO_TOKEN=vKN3jjhAOwxkv9HG0VBX4EYIlWLPwiJ9d npm test` above ^^^